### DISCUSSION TITLE
New file - access rights

### DISCUSSION THREAD

**Discussion by mcp1766** [phcode-dev/phoenix#1198] [Q&A]
Hey,

When opening a local folder and create a new file it gets -rw------- attributs.  When opening it in the browser I receive access denied.  Need to change it and add o+rw to get access.  How can I change the defaut permissions when creating a new file.

![Screenshot_20231122_105937](https://github.com/phcode-dev/phoenix/assets/27546954/21f2b0d5-7b25-4c2a-adee-d81ce98f5d64)

**Comment by charlypa** [comment]
Are you creating files through phcode or locally
whats the operating system?

**Reply by mcp1766** [reply]
Through phcode.  It is Manjaro and Chromium.  On my local folder I need to : `chmod o+rw` on the file to gain access right .  As you can see on the picture "test3.html" just after creating the file tehn I receive from my localhost Access denied 403.

![access-right1](https://github.com/phcode-dev/phoenix/assets/27546954/6d53ecb7-9919-4815-9f41-5f3b54011727)

**Reply by charlypa** [reply]
I tried the same with the latest Manjaro on a virtual machine. I am able to read and write files to my folder. 
would you be able to share the console errors to debug further
